redash/settings/__init__.py

```diff
@@ -312,7 +312,13 @@ def email_server_is_configured():
 )
 
 ALERTS_DEFAULT_MAIL_SUBJECT_TEMPLATE = os.environ.get(
-    "REDASH_ALERTS_DEFAULT_MAIL_SUBJECT_TEMPLATE", "({state}) {alert_name}"
+    "REDASH_ALERTS_DEFAULT_MAIL_SUBJECT_TEMPLATE",
+    "Alert: {alert_name} changed status to {state}"
+)
+
+REDASH_ALERTS_DEFAULT_MAIL_BODY_TEMPLATE = os.environ.get(
```

Comment:
It might be clearer to call it "REDASH_ALERTS_DEFAULT_MAIL_BODY_TEMPLATE_FILE" to indicate that it is a file path.

Reply:
done